### DISCUSSION TITLE
dynamic items unavailable in current release (2.3.2)

### DISCUSSION THREAD

**Discussion by lukvmil** [Rapptz/discord.py#9700] [General] [answered]
I'm using the dynamic item UI elements in a project and noticed they aren't actually supported in the current release. The code for them is there, and there is an official example using them, but dyanmic.py is not included in the ui module's __init__.py, so it cannot be imported and used as shown in the example. Is this intentional or is this a bug? I've been having to pip install from the current repository head on GitHub, but this isn't necessarily stable and it would be nice to see it in an official release.

**Comment by No767** [comment, marked as answer]
It's not in the 2.3.2 release. It's available on the master branch, which you can install using git. Iirc there are some blockers that are preventing the release of 2.4.0 (I may be incorrect on this)

**Reply by lukvmil** [reply]
ah ok thank you, I see that the dynamic item example is not in the 2.3.2 release so that makes sense
